src/it/JENKINS-45740-metadata/verify.groovy

```diff
@@ -0,0 +1,37 @@
+import java.util.jar.*
+
+assert new File(basedir, 'target/JENKINS-45740-metadata.hpi').exists()
+
+File installed = new File(basedir, '../../local-repo/org/jenkins-ci/tools/hpi/its/JENKINS-45740-metadata/1.0-SNAPSHOT/')
+assert installed.directory
+
+def checkArtifact(File installed, String artifact, Map<String,String> expectedManifestEntries) {
+    File f = new File(installed, artifact)
+    assert f.file
+    JarFile j = new JarFile(f)
+    try {
+        Attributes attr = j.manifest.mainAttributes
+        expectedManifestEntries.each {header, value -> assert attr.getValue(header) == value}
+    } finally {
+        j.close()
+    }
+}
+
+checkArtifact(
+    installed,
+    'JENKINS-45740-metadata-1.0-SNAPSHOT.hpi',
+    [
+      'Plugin-ChangelogUrl': 'http://plugins.jenkins.io/JENKINS-45740-metadata/changelog',
+      'Plugin-LogoUrl': 'http://plugins.jenkins.io/JENKINS-45740-metadata/logo.png',
+      'Plugin-ScmUrl': 'http://github.com/jenkinsci/JENKINS-45740-metadata-plugin',
+      'Plugin-License-Name-1': 'MIT License',
+      'Plugin-License-Url-1': 'https://opensource.org/licenses/MIT',
+      'Plugin-License-Distribution-1': 'repo',
```

Comment:
I have never even understood what this field of the POM is for, but I think we do not need it. Really I would suggest dropping the name too, and only support _canonical_ URLs: `https://opensource.org/licenses/.+`